### PR TITLE
1944 feat info palette

### LINKED ISSUE #1944: [Feature?]: (Docs) Add Info Palette reference page
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

Create an info palette reference page similar to #1926, which was for the neutral palette. Should be based off of [Figma UI Kit](https://www.figma.com/proto/uJtPfI38D9i8iQg0UGK2E0/Pando-Design-Guidelines?node-id=481-19105&starting-point-node-id=6%3A11626)

### Examples 🌈

`/reference/colors/neutral-palette` in new docs

### Motivation 🔦

Each component of the color palette will need its own page for reference.

### PR DESCRIPTION
**Pull request by kathydlavoie-ps**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Closes #1944

## What is the new behavior?

* Adds new info palette page (migrated over from [figma](https://www.figma.com/proto/uJtPfI38D9i8iQg0UGK2E0/Pando-Design-Guidelines?node-id=481-19105&starting-point-node-id=6%3A11626))
* Adds navigation to info palette page

### CODE REVIEW

File: docs/src/content/docs/reference/color/info-palette.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+<ColorSwatch
+  title="text-initial"
+  description="Default text inside of status elements"
+  uses="Admonition text, Toast text, Badges"
+  colorTokenName="info.text.initial"
+>
+  <div class={css(colorSwatchStyles, { bg: 'info.text.initial' })} />
+</ColorSwatch>
+
+:::caution[Beta]
```
Comment (kathydlavoie-ps) [inline docs/src/content/docs/reference/color/info-palette.mdx:26]: This is migrated over from the figma docs. I've made a note to figure out more descriptive messaging 👍

File: docs/src/content/docs/explanation/color.md [hunk 2]
```diff
@@ -13,18 +13,20 @@ Every color should convey a purpose, which is its sentiment
 
 A palette for non-actionale items, eg. page content
 
-<a href='/reference/colors/neutral-palette'>view more about neutral colors</a>
+<a href='/reference/color/neutral-palette'>view more about neutral colors</a>
```
Comment (caseybaggz) [inline docs/src/content/docs/explanation/color.md:16]:
```suggestion
<a href='/reference/color/neutral-palette'>View the neutral color reference docs</a>
```

File: docs/src/content/docs/reference/color/info-palette.mdx (hunk 1 above)
Comment (caseybaggz) [reply docs/src/content/docs/reference/color/info-palette.mdx:26]: I'm not seeing a reason to include this. The designs don't use over half the tokens we have so why call out every single one?

Comment (kathydlavoie-ps) [reply docs/src/content/docs/reference/color/info-palette.mdx:26]: Do you mean you are not seeing a reason not to include the admonition? It was just a solution to capture the messaging in the figma doc 🤷 in conversation with @bradryanbice it seems as though we may just remove all of these "non-recommended" color tokens. We will prioritize making a decision here 👍

**Review by caseybaggz — APPROVED** [review]
🚢

**Pull request merged**